doc/examples/filters/plot_frangi.py

```diff
@@ -0,0 +1,29 @@
+"""
+==============
+Frangi filter
+==============
+
+Frangi and hybrid Hessian filters can be used for edge detection and
+calculation of fraction of the image containing edges.
+"""
+
+from skimage.data import camera
+from skimage.filters import frangi, hessian
+
+import matplotlib.pyplot as plt
+
+
+image = camera()
+
+fig, ax = plt.subplots(ncols=2, subplot_kw={'adjustable':'box-forced'})
+
+ax[0].imshow(frangi(image), cmap=plt.cm.gray)
```

Comment:
@sciunto I've taken over the PR, see #2153. I'll make those changes. Closing this one now. Thank you @Hiyorimi!